src/resources/ui/preferences.ui

```diff
@@ -43,6 +43,18 @@
                         </property>
                     </object>
                 </child>
+
```

Comment:
You need to lock idle threshold settings if `invert-running-speed` is enabled.

Example: https://github.com/aunetx/blur-my-shell/blob/master/resources/ui/dash.ui#L53
Ping me if you need an assistance with this

Reply:
Good idea. Before you could change the idle threshold setting but nothing would happen.

> <property name="sensitive" bind-source="blur" bind-property="state" bind-flags="sync-create" />
This didn't work, cause i needed the inverted the behavior.